lib/sql_string.js

```diff
@@ -130,11 +122,11 @@ function zeroPad(number, length) {
 }
 
 function convertTimezone(tz) {
-  if (tz == "Z") return 0;
+  if (tz === "Z") return 0;
 
   var m = tz.match(/([\+\-\s])(\d\d):?(\d\d)?/);
   if (m) {
-    return (m[1] == '-' ? -1 : 1) * (parseInt(m[2], 10) + ((m[3] ? parseInt(m[3], 10) : 0) / 60)) * 60;
+    return (m[1] === '-' ? -1 : 1) * (parseInt(m[2], 10) * 60 + (m[3] ? parseInt(m[3], 10) : 0));
   }
-  return false;
+  return 0;
```

Comment:
I can't remember if there is anything specific to this module here in sql_string ( except from file name convention change ) We started discussion with @dougwilson about factoring out some common code own packages and moving some shared code to https://github.com/mysqljs org. SqlString might be good first candidate.